lib/sinon/util/core/deprecated.js

```diff
@@ -0,0 +1,24 @@
+/*eslint no-console: 0 */
+"use strict";
+
+// wrap returns a function that will invoke the supplied function and print a deprecation warning to the console each
+// time it is called.
+exports.wrap = function (func, msg) {
+    return function () {
+        // Watch out for IE7 and below! :(
+        if (typeof console !== "undefined") {
+            if (console.warn) {
```

Comment:
Ah sorry I have not pushed the changes mentioned above as I was waiting on
the sandbox cjs work to land first.

On Wed, 27 Jul 2016, 23:33 Carl-Erik Kopseng, <notifications@github.com>
wrote:

> In lib/sinon/util/core/deprecated.js
> <https://github.com/sinonjs/sinon/pull/1090#discussion_r72536795>:
>
> > @@ -0,0 +1,24 @@
> > +/*eslint no-console: 0 */
> > +"use strict";
> > +
> > +// wrap returns a function that will invoke the supplied function and print a deprecation warning to the console each
> > +// time it is called.
> > +exports.wrap = function (func, msg) {
> > +    return function () {
> > +        // Watch out for IE7 and below! :(
> > +        if (typeof console !== "undefined") {
> > +            if (console.warn) {
>
> Not seeing much trace of the console.info instead of console.warn thing
> you talked about for PhantomJS?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sinonjs/sinon/pull/1090/files/131555150b9b8785bea6923702d7102e3fe9732f#r72536795>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAMN-edIe5DVMCZV_IBMnUSuJjIdGk3Qks5qZ9yYgaJpZM4JI4nR>
> .
>